assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js

```diff
@@ -36,16 +36,40 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import styled from 'styled-components';
 import MediaGallery from '../common/mediaGallery';
 import {
   MediaGalleryContainer,
   MediaGalleryInnerContainer,
   MediaGalleryLoadingPill,
   MediaGalleryMessage,
 } from '../common/styles';
+import { ReactComponent as UnsplashLogoFull } from '../../../../../icons/unsplash_logo_full.svg';
+import theme from '../../../../../theme';
+import { ProviderType } from './providerType';
 
 const ROOT_MARGIN = 300;
 
+const AttributionPill = styled.div`
+  position: absolute;
+  left: 24px;
+  bottom: 10px;
+  border-radius: 100px;
+  padding: 5px 8px;
+  line-height: 16px;
+  display: flex;
+  flex-wrap: nowrap;
+  font-size: 12px;
```

Comment:
Shall the font size, color and line-height come from the theme?

Reply:
I see the advantage of using the theme for the background-color and color, but I question what the value is for the font size, line-height, etc. The theme should provide semantically charged configurations that allow us to switch eventually between themes (dark and light mode). In this case what would be the advantage to create a new attribution font size and line height in the theme when it won't be used anywhere else in the application?

Reply:
I also see the biggest value in the colors. But other places also make use of it for sizes. I think it makes it easier to be consistent and see all resources in one place. It is done e.g. similarly in Android. But I'm not strong about it.

Reply:
For example, there is no rgba(0,0,0,0.7) in the theme. Am I supposed to create a new 
```
media3p: {
  attribution: {
    bg: rgba(0,0,0,0.7)
  }
}
```
in there, or simply add a new bg color like
```
colors: {
  bg: {
    v16: rgba(0,0,0,0.7)
  }
}
```
It's very unclear how themes are meant to be used IMO.

Reply:
In this case the font-size and line-height has been fine tuned to match the height of the unsplash SVG logo's font, so I'd rather leave them as hardcoded values in the css.

Reply:
`media3p.attribution` would have been fine.